ELIXIR.md

```diff
@@ -83,7 +83,7 @@ The project's `mix.exs` file should include hex.pm package and description attri
 
 ### Travis.yml
 
-Specially for open-source projects integrating open code checkers with Travis CI builds, the `.travis.yml` file in your project root folder should look similar to:
+Especially for open-source projects integrating open code checkers with Travis CI builds, the `.travis.yml` file in your project root folder should look similar to:
```

Comment:
I found 2 misspelled words in this sentence: 
- CI
	- *Suggested Replacements:* cm, cr, fi, cfi, cie, ti, ca, ki, cf, bi, cit, ck, yi, cp, pi, cpi, c, ic, cu, gi, ni, co, ui, ct, cd, li, cir, cli, cy, si, cig, csi, sci, pci, cs, cc, ai, cia, cai, ch, xi, hi, civ, cif, chi, cb, vi, cg, mi, cl, di, cq, cv, cid, i, ii, ce, wi, cis
- open-source